score_models/sde/sde.py

```diff
@@ -1,48 +1,66 @@
 from abc import ABC, abstractmethod
-from typing import Tuple
+from typing import Optional
 
 import torch
 from torch.distributions import Normal, Independent
-from torch.distributions import Distribution
 from torch import Tensor
+from score_models.utils import DEVICE
+
 
 class SDE(ABC):
     """
     Abstract class for some SDE info important for the score models
     """
-    def __init__(self, T=1.0, epsilon=0., **kwargs):
+
+    def __init__(self, t_min=0.0, t_max=1.0, **kwargs):
         """
-        The time index in the diffusion is defined in the range [epsilon, T]. 
+        The time index in the diffusion is defined in the range [t_min, t_max].
```

Comment:
Some saved models have epsilon defined as their hyperparameter. To make this change, I would like to have backward compatibility with older code to make sure score models saved before this change can still be loaded without errors. In essence, we need to change the loading function to make sure parameters are translated from epsilon to t_min.

Reply:
Ok, I think I updated it so that epsilon and t_min are stored in hyperparameters. Not sure if that is enough for backwards compatability?